Paws4Me/Paws4Me/Controllers/SignInViewController.swift

```diff
@@ -12,47 +12,37 @@ class SignInViewController: UIViewController {
     @IBOutlet weak var txtPassword: UITextField!
     @IBOutlet weak var btnSignIn: UIButton!
     @IBOutlet weak var btnRegister: UIButton!
-    
     var bLoggedIn = false
-    
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         btnSignIn.layer.cornerRadius = 10
         btnRegister.layer.borderWidth = 1.0
         btnRegister.layer.cornerRadius = 10
-        btnRegister.layer.borderColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
+        btnRegister.layer.borderColor = UIColor.init(red: 55/255, green: 21/255, blue: 67/255, alpha: 1).cgColor
```

Comment:
This can be added to the style kit 😄 also button styling can be done in an extension.